SU2_CFD/src/variables/CIncEulerVariable.cpp

```diff
@@ -90,7 +91,7 @@ CIncEulerVariable::CIncEulerVariable(su2double pressure, const su2double *veloci
 
   Primitive.resize(nPoint,nPrimVar) = su2double(0.0);
 
-  /*--- Incompressible flow, gradients primitive variables nDim+4, (P, vx, vy, vz, T, rho, beta) ---*/
+  /*--- Incompressible flow, gradients primitive variables nDim+6, (P, vx, vy, vz, T, rho, beta, lamMu, EddyMu) ---*/
```

Comment:
insert emoji with monocle

Reply:
🧐 there you go

Reply:
Where the heck do you get them from?

Reply:
`:monocle_face`

Reply:
most of the time I just put the `:` and type want I want to have (crazy, right :) ) those github emoji cheatsheets are often incomplete

Reply:
:facepalm: here I was just trying to scroll down after typing :